.changeset/lucky-falcons-dress.md

```diff
@@ -0,0 +1,39 @@
+---
+'@backstage/cli': minor
+'@backstage/create-app': minor
+---
+
+The analytics plugin removed the need for app.googleAnalyticsTrackingId and the script tags in packages/app/public/index.html.
```

Comment:
Could be a good opportunity to link off to the GA module's installation instructions specifically.  Could replace this line with something like...

> The [Analytics API](https://backstage.io/docs/plugins/analytics) is the recommended way to track usage in Backstage; an optionally installable [Google Analytics module](https://github.com/backstage/backstage/tree/master/plugins/analytics-module-ga#installation) has superseded the old `app.googleAnalyticsTrackingId` config and its corresponding script tags in `packages/app/public/index.html`. 

My main concern is that someone sees this in the upgrade instructions and removes the config/tags without actually having installed the replacement.

Reply:
I have updated the changeset to include this verbiage and links. Thank you!